src/validators/claims-validator.ts

```diff
@@ -27,3 +28,21 @@ export const areClaimsValid = (claims: string[], config: Config): boolean => {
 
   return true;
 };
+
+export const checkForReturnCode = (
+  config: Config,
+  claims: string[],
+  redirectUri: string
+): void => {
+  const returnCode = config.getReturnCode();
+  if (returnCode) {
```

Comment:
could you do a falsy check and return if falsy, and then do a check on the claims? avoid the nested if statement